### PR TITLE
13- Done- Vinz

### PR DESCRIPTION
**Pull request by vinc86**

### CODE REVIEW

File: 13-PB-language-boolean/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+// task 1
+console.log("Task 1:");
+console.log(
+  `- 3="3" output: SyntaxError: Invalid left-hand side in assignment`
+); // returns an error because a single = is used to assign values into declared variables/constants
+
+console.log(`- 3 == "3" output: ${3 == "3"}`); // return true, it checks only the value
+console.log(`- 3 === "3" output: ${3 === "3"}`); // return false, it checks value and type, and they are two different types (number and string)
+
```
Comment (Ungoot) [inline 13-PB-language-boolean/index.js:11]: Please answer this question too;
Which comparison operator should we generally use? Why?

**Comment by vinc86** [comment]
Oh Shure.. i haven't seen it 😅

Il mar 26 mag 2020, 09:51 Bleda <notifications@github.com> ha scritto:

File: 13-PB-language-boolean/index.js (hunk 1 above)
Comment (vinc86) [reply 13-PB-language-boolean/index.js:12]: done! thanks!

**Comment by Ungoot** [comment]
Good job

**Pull request closed without merging**